daemon/MenuDaemon.vala

```diff
@@ -258,5 +268,65 @@ namespace Gala {
                 push_in = true;
             }, 3, Gdk.CURRENT_TIME);
         }
+
+        private void init_desktop_menu () {
+            change_wallpaper_accellabel = new Granite.AccelLabel (_("Change Wallpaper…"));
```

Comment:
Since none of these have accelerators, you could probably just do `Gtk.MenuItem.MenuItem.with_label`